static/js/app.js

```diff
@@ -256,7 +256,7 @@ $(function () {
 
             _.each(this.oms.markersNearAnyOtherMarker(), function (marker) {
                 // hide markers using markers.css
-                marker.title = '';
+                marker.view.model.get("currentlySpiderfied")? marker.setTitle(marker.view.getTitle("single")) : marker.setTitle(marker.view.getTitle("multiple"));
                 var groupHead = marker.view.model;
```

Comment:
It's not the group "head" anymore so there's no reason for this name.
In the previous implementation, only the first marker in each group got the special title.
This is related to #481, because if there are several partially transparent on top of each other, then their opacities are summed. So make sure it's correct to do this for all markers in the group.